src/services/database/migrate.ts

```diff
@@ -1,16 +1,23 @@
 import { config } from "dotenv";
 config({ path: ".env.local" });
 import { migrate as migrateVercel } from "drizzle-orm/vercel-postgres/migrator";
-import { migrate as migrateNode } from "drizzle-orm/node-postgres/migrator";
+import { migrate as migrateNode } from "drizzle-orm/postgres-js/migrator";
 import { db } from "@/services/database/connection";
+import { VercelPgDatabase } from "drizzle-orm/vercel-postgres";
+import * as schema from "./schema";
+import { PostgresJsDatabase } from "drizzle-orm/postgres-js";
 
 async function main() {
   console.log("Running migrations...");
 
   if (process.env.NODE_ENV === "production") {
-    await migrateVercel(db, { migrationsFolder: "drizzle/migrations" });
+    await migrateVercel(db as VercelPgDatabase<typeof schema>, {
+      migrationsFolder: "drizzle/migrations",
+    });
   } else {
-    await migrateNode(db, { migrationsFolder: "drizzle/migrations" });
+    await migrateNode(db as PostgresJsDatabase<typeof schema>, {
+      migrationsFolder: "drizzle/migrations",
```

Comment:
According to docs it is recommended to use `max: 1` for connection pool size for migrations
https://orm.drizzle.team/docs/get-started-postgresql